runtime/redirector/generated.c.m4

```diff
@@ -332,8 +243,8 @@ JVM_LoadSystemLibrary(const char *libName)
  * com.sun.tools.attach.VirtualMachine support
  *
  * Initialize the agent properties with the properties maintained in the VM.
- * The following properties are apparently set by the reference implementation:
- *  sun.java.command = name of the main class
+ * The following properties are apparently set by the Sun implementation:
```

Comment:
Was there a problem with the word "reference"? I don't think "Sun" should be used. If its necessary to change, please use "OpenJDK".